mautrix_facebook/portal.py

```diff
@@ -544,7 +555,7 @@ def _get_facebook_reply(self, reply: str) -> Optional[RelatesTo]:
                                          relates_to=self._get_facebook_reply(reply_to))
 
     async def _handle_facebook_attachment(self, intent: IntentAPI, attachment: AttachmentClass,
-                                          reply_to: str) -> Optional[EventID]:
+                                          reply_to: str, message_urls: list) -> Optional[EventID]:
```

Comment:
The type hint for lists is `List[item type]` (with `from typing import List`)

Reply:
changed `message_urls: list` to `message_text: str`
Good to know though, I never really used python's type hints :)